kairon/chat/routers/voice.py

```
@@ -0,0 +1,51 @@
+from kairon.shared.models import User
+
+router = APIRouter()
+
+_EMPTY_TWIML = "<?xml version='1.0' encoding='UTF-8'?><Response/>"
+
+_HEADERS = {"Content-Type": "text/xml; charset=utf-8"}
+
+@router.post("/{bot}/channel/voice/{provider}/call/{token}")
+async def handle_incoming_voice_call(
+        request: Request,
+        bot: Text = Path(description="Bot id"),
```

Ruff (0.15.12)

[warning] Do not perform function call `Path` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)